chain-core/src/init/config.rs

```diff
@@ -42,6 +44,18 @@ pub struct JailingParameters {
     pub missed_block_threshold: u16,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Encode, Decode)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct SlashingParameters {
+    /// Percentage of funds (bonded + unbonded) slashed when validator is not live (liveness is calculated by jailing
+    /// parameters)
+    pub liveness_slash_percent: u8,
+    /// Percentage of funds (bonded + unbonded) slashed when validator makes a byzantine fault
+    pub byzantine_slash_percent: u8,
+    /// Number of blocks to wait before slashing funds from an account
+    pub slash_wait_blocks: u32,
```

Comment:
`unbonding_period` is in unix time -- perhaps make it consistent and check the value <= `unbonding_period`: https://github.com/cosmos/cosmos-sdk/blob/master/x/slashing/spec/07_tombstone.md
ultimately the processing / slash period tracking-execution could be similar https://github.com/cosmos/cosmos-sdk/blob/master/x/slashing/spec/04_begin_block.md except that there's no delegation